src/electron/views/device-connect-view/components/device-connect-port-entry.tsx

```diff
@@ -40,24 +42,24 @@ export class DeviceConnectPortEntry extends React.Component<
     public render(): JSX.Element {
         return (
             <div className={styles.deviceConnectPortEntry}>
-                <h2>Android device port number</h2>
+                <div id={textFieldLabelId} className={styles.portNumberLabel}>
```

Comment:
Consider using `<label for="text-field-id">` rather than the div here; it's more semantically accurate and allows for extra native browser features (eg, it causes clicking on the label to count as focusing the associated input - give it a try at https://codepen.io/dbjorge/pen/WNvEQrV )

Reply:
Great suggestion. Updated.